extensions/omni_httpc/omni_httpc.c

```diff
@@ -317,6 +317,16 @@ static h2o_httpclient_head_cb on_connect(h2o_httpclient_t *client, const char *e
   *_method = req->method;
   *num_headers = req->num_headers;
   *headers = req->headers;
+  /*
+  required to prevent malformed host header with unix domain sockets
+  for eg. when calling http://[unix:/var/run/docker.sock]/_ping
+  h2o sets the host header to [unix:/var/run/docker.sock] and golang
+  disallows '/' in host header value with 400 Bad request
```

Comment:
Should we include a reference to a formal (or informal) description of the syntax here or in the docs?

Reply:
I will add description here, the syntax I believe is h2o specific so i suggest we introduce some sort of abstraction before introducing unix domain sockets to the end user